docs/user-guide/nextcloud-integration/README.md

```diff
@@ -69,7 +69,61 @@ In addition to listing files attached to a work package, the **Files** tab now a
 
 ![OP_linked_files](1_1_00-All_files_available.png)
 
-> **Note:** In this version of the integration, you can only link files to work packages on Nextcloud; adding a new link to a Nextcloud file via the OpenProject interface is not yet possible, but will be possible in the near future.
+
+To link a Nextcloud file to the current work package, you can either:
+
+- select a file from your computer, which will be uploaded to Nextcloud and linked to this work package
+- select an existing file in Nextcloud to link to
+
+#### Upload and link file(s)
+
+If the file you want to link has not yet been uploaded to Nextcloud, you can do so by clicking on the **Upload files** link. 
+
+>> IMG: Focus on Upload file link
+
+You will then be prompted to select a file (or multiple files) on your computer that you want to upload to Nextcloud. 
+
+>> IMG: Local browser file picker
+
+Alternatively, you can also simply drag a file or folder on your computer on to this area (under the name of your Nextcloud file storage) and drop it in the drop zone that appears.
+
+Once you have selected or dropped the files you would like to upload, you will need to select a folder on Nextcloud to which they should be stored. 
+
+> **Info:** This step allows you to follow any guidelines your organisation might have concerning folder structure or file management, and where new files should be stored.
+
+You can click on folders you see to navigate to them. A helpful breadcrumb shows you where you are in the folder hierarchy. 
+
+To navigate one level up or to go back to the root, simply click on the relevant parent in the breadcrumb. 
+
+> **Info**: If you have navigated particularly deep (over 4 levels), intermediate levels might be collapsed to save space, but you'll always be able to navigate back to the immediate parent or the root to go backwards.
+
+To save the files you uploaded to the currently open folder, click on the **Choose location** button.
+
+#### Link existing files
+
+If the file you want to link already exists on Nextcloud, you simply have to create the link.
+
+To do so, start by clicking on **Link existing file**.
+
+A file picker will appear, displaying all the files and folders on your Nextcloud instance that you have permission to see.
+
+>> IMG: Nextcloud file picker
+
+To link a certain file or folder with this work package, click on the checkbox to the left of each item.
+
+>> IMG: Certain files and folder checked
+
+If the file you're looking for is inside a folder or multiple levels of folders, you can click on each one to navigate to it. A helpful breadcrumb shows you where you are in the folder hierarchy. 
+
+To navigate one level up or to go back to the root, simply click on the relevant parent in the breadcrumb. 
+
+> **Info**: If you have navigated particularly deep (over 4 levels), intermediate levels might be collapsed to save space, but you'll always be able to navigate back to the immediate parent or the root to go backwards.
```

Comment:
@psatyal do we need this info? Also we are repeating it in both sections.

Reply:
Althoough this behaviour _should_ be obvious and easy to understand, I think it's better to mention it in the documentation, just in case it isn't. I repeat that bit on purposes, since someone might only read the bit that concerns them. (I assume users won't read all sections if there are two ways of doing the the same or a similar thing).